lib/node_modules/@stdlib/lapack/base/dpttrf/README.md

```diff
@@ -0,0 +1,252 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dpttrf
+
+> Compute the `L * D * L ** T` factorization of real symmetric positive definite tridiagonal matrix `A`.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dpttrf = require( '@stdlib/lapack/base/dpttrf' );
+```
+
+#### dpttrf( N, D, E, info )
+
+Computes the `L * D * L ** T` factorization of real symmetric positive definite tridiagonal matrix `A`.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var N = 3;
+var info = 0;
+var D = new Float64Array( [ 4.0, 5.0, 6.0 ] );
+var E = new Float64Array( [ 1.0, 2.0 ] );
+
+dpttrf( N, D, E, info );
+// D => <Float64Array>[ 4, 4.75, ~5.15789 ]
+// E => <Float64Array>[ 0.25, ~0.4210 ]
+```
+
+The function has the following parameters:
+
+-   **N**: order of matrix `A`.
+-   **D**: the N diagonal elements of `A` as a [`Float64Array`][mdn-float64array].
+-   **E**: the N-1 subdiagonal elements of `A` as a [`Float64Array`][mdn-float64array].
+-   **info**: status code, `0` if successful, `<0` if i-th argument had an illegal value, `>0` if leading principal minor of order `i` is not positive; if `<N` then factorization could not be completed, if `>N` then the factorization was completed, but `D(N) <= 0`.
```

Comment:
And actually, in JavaScript, we shouldn't have an `info` parameter. This only makes sense when you can pass by reference. In any JS invocation, this will be passed by value. This parameter should be removed and the list of possible return values moved to the `Notes` section below.

Reply:
Yes, makes sense. Updated it.